### PR TITLE
getting data after page rendering

### PR DESCRIPTION
**Pull request by dacolgit**
add spinner when loading data
optimized read value when browsing
fix scan status
'about' button redirect to github Industrial-Iot page

### CODE REVIEW

File: samples/src/Microsoft.Azure.IIoT.App/src/Pages/Applications.razor [hunk 1]
```diff
@@ -54,11 +56,27 @@
...
+    protected override void OnInitialized()
+    {
+        _spinner = "loader-big";
+    }
+
+    /// <summary>
+    /// OnAfterRenderAsync
+    /// </summary>
+    /// <param name="firstRender"></param>
+    protected override async Task OnAfterRenderAsync(bool firstRender) {
```
Comment (marcschier) [inline samples/src/Microsoft.Azure.IIoT.App/src/Pages/Applications.razor]: Let's follow consistent formatting.

File: samples/src/Microsoft.Azure.IIoT.App/src/Pages/Discoverers.razor [hunk 2]
```diff
@@ -151,50 +157,67 @@
...
+
+    /// <summary>
+    /// OnAfterRenderAsync
+    /// </summary>
+    /// <param name="firstRender"></param>
+    protected override async Task OnAfterRenderAsync(bool firstRender)
+    {
+        if (firstRender)
+        {
+            discovererList = await RegistryHelper.GetDiscovererListAsync();
```
Comment (marcschier) [inline samples/src/Microsoft.Azure.IIoT.App/src/Pages/Discoverers.razor]: Let's use query with paging.

Comment (dacolgit) [reply samples/src/Microsoft.Azure.IIoT.App/src/Pages/Discoverers.razor]: I'm already using query with paging

Comment (marcschier) [reply samples/src/Microsoft.Azure.IIoT.App/src/Pages/Discoverers.razor]: So where are you calling the API to get the next page?   I can see you use query, but what I am missing is where you query again for the next page.   We should use the paging the API provides, not query all and then page the list, the former is far more efficient - also for all the other API, including browsing.

**Pull request merged**